awx/ui_next/jest.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     '\\.(css|scss|less)$': '<rootDir>/__mocks__/styleMock.js',
     '^@api(.*)$': '<rootDir>/src/api$1',
     '^@components(.*)$': '<rootDir>/src/components$1',
+    '@constants$': '<rootDir>/src/constants.js',
```

Comment:
Should this start with `^` like the other path shortcuts?

Reply:
Yea - it's safer that way

Reply:
Should now be fixed